lightning/src/routing/router.rs

```diff
@@ -1076,7 +1076,7 @@ impl_writeable_tlv_based!(RouteParametersConfig, {
 
 impl RouteParametersConfig {
 	/// Initates an new set of route parameter configs with default parameters.
-	pub fn new() -> Self {
+	pub fn default() -> Self {
```

Comment:
May as well have this implement the `Default` trait instead.

Reply:
Thanks for the pointer!
Addressed in [pr3342.12](https://github.com/shaavan/rust-lightning/commits/pr3342.12)